pull_request_template.md

```diff
@@ -20,8 +20,16 @@
 - Go to the [contribution guidelines](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md) and read it again.
 
 
-RULES (because you probably didn't read the guidelines): 
-* ORDER: The entries are in lexical order (English letter order) 
-* CREDIT: Each entry must contains the name of the author with a link to its github account!
-* LINK: the link should be to the source code (https://github/account/project) not the website (https://myproject.io <- no!)
-* MONEY: indicates if one needs a paying subscription to use that project you could just add ($$$) next to the name ([my project - $$$] (link to my project) )
+**RULES (because you probably didn't read the guidelines):**
+- FORMAT
+  - Use the following format: `[Name](Link) - Description.`
+  - Name should be the title of the project
+  - Link should be to the GitHub source code, not to the website
```

Comment:
You're right, it sounds bit weird. Maybe like this?
- Link should be a GitHub repository, not a website